qCC/ccCommandLineCommands.h

```diff
@@ -448,6 +450,60 @@ struct CommandLoad : public ccCommandLineInterface::Command
 	}
 };
 
+struct CommandOctreeNormal : public ccCommandLineInterface::Command
+{
+	CommandOctreeNormal() : ccCommandLineInterface::Command("OCTREE_NORMALS", COMMAND_COMPUTE_OCTREE_NORMALS) {}
+
+	virtual bool process(ccCommandLineInterface& cmd) override
+	{
+		cmd.print("[OCTREE NORMALS CALCULATION]");
+		ccLog::Error("OCTREE NORMALS CALCULATION");
+		if (cmd.clouds().empty())
+		{
+			return cmd.error(QString("No point cloud to normal calculation (be sure to open one with \"-%1 [cloud filename]\" before \"-%2\")").arg(COMMAND_OPEN).arg(COMMAND_SUBSAMPLE));
+		}
+
+		if (cmd.arguments().empty())
+		{
+			return cmd.error(QString("Missing parameter: calculation method after \"-%1\"").arg(COMMAND_COMPUTE_OCTREE_NORMALS));
+		}
+
+		bool ok;
+		float radius = cmd.arguments().takeFirst().toFloat(&ok);
+		if (!ok)
+		{
+			return cmd.error(QString("Missing radius parameter"));
+		}
+
+		cmd.print(QString("\tRadius: %1").arg(radius));
+
+		CC_LOCAL_MODEL_TYPES model = QUADRIC;
+		ccNormalVectors::Orientation  orientation = ccNormalVectors::Orientation::UNDEFINED;
+		PointCoordinateType defaultRadius = radius;
+		for (size_t i = 0; i < cmd.clouds().size(); ++i)
+		{
+			ccPointCloud* cloud = cmd.clouds()[i].pc;
+			cmd.print("computeNormalsWithOctree started...\n");
+			bool success = cloud->computeNormalsWithOctree(QUADRIC, orientation, defaultRadius, nullptr);
+			if(success){
```

Comment:
This is a formatting issue. While you're fixing this, maybe fix indentation on the "for"?